components/zoho_books/common/utils.mjs

```
@@ -0,0 +1,41 @@
+
+export const clearObj = (obj) => {
+  return Object.entries(obj)
+    .filter(([
+      ,
+      v,
+    ]) => (v != null && v != "" && JSON.stringify(v) != "{}"))
+    .reduce((acc, [
+      k,
+      v,
+    ]) => ({
+      ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)